src/entities/notefolder.cpp

```diff
@@ -530,12 +530,12 @@ bool NoteFolder::isPathNoteFolder(const QString &path) {
 
 void NoteFolder::setSettingsValue(const QString &key, const QVariant &value) {
     SettingsService settings;
-    settings.setValue(QString("NoteFolder-%1/%2").arg(QString::number(id), key), value);
```

Comment:
Did you test the code? 😅 This breaks the function.

Reply:
Oh, sorry :(

Reply:
At least it started to throw warnings and I have a bad feeling, can you please stick to just replacing the QString. 😉

Reply:
So... are failed tests due to this change?

Reply:
There are no tests for testing if your change breaks the string. But it seems ok. Thank you.